tekton/README.md

```diff
@@ -92,7 +92,10 @@ Install Task from plumbing too:
 # Apply the Tasks we are using from the catalog
 kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/golang-build/0.3/golang-build.yaml
 kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/golang-test/0.2/golang-test.yaml
-kubectl apply -f https://raw.githubusercontent.com/tektoncd/plumbing/main/tekton/resources/release/
+
+# Apply Tasks and other resources from Plumbing
+# Run this command from the root of the plumbing repo
+kubectl kustomize ./tekton/resources/release | kubectl apply -f -
```

Comment:
plumbing covers both namespaces `default` and `tekton-nightly` and without the namespace created before running this command, it fails with:

```
Error from server (NotFound): error when creating "STDIN": namespaces "tekton-nightly" not found
```

Should we include a pre-req here to create the required namespace?

Reply:
Thanks @pritidesai ! Updated with expanded comment describing this requirement.

Reply:
An alternative would be to install only the `default` overlay
```
kubectl kustomize ./tekton/resources/release/overlay/default
```

Reply:
Updated with this alternative!